src/redux/actions/userDetails/index.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
+import { resolveOrders } from '../../services/orders';
+import resolveUserDetails from '../../services/userDetails';
+
+export const STOP_LOADING_ORDERS = 'STOP_LOADING_ORDERS';
+export const LOAD_USER_DETAILS = 'LOAD_USER_DETAILS';
+
+export default function fetchUserDetailsData(uid) {
+    return (dispatch, getState) => {
+        const { isFirstTime } = getState().userDetails;
+        if (isFirstTime) {
+            const promises = [
+                resolveOrders(uid),
```

Comment:
this must be another service resolveUserOrders that goes inside orders services folder
- resolveOrders resolve all orders for all users.
- resolveUserOrders resolve only user specific orders.